src/runner.rs

```diff
@@ -1109,11 +1110,7 @@ impl FunctionContext {
 			.map(RuntimeValue::default)
 			.collect::<Vec<_>>();
 
-		// TODO: Replace with extend.
-		for local in locals {
-			value_stack.push(local)
-				.map_err(|_| TrapKind::StackOverflow)?;
-		}
+		value_stack.extend(&locals)?;
```

Comment:
I wonder if this TODO is actually [still] relevent. Have you performed benchmarks before/after?

Reply:
Not  yet, I assume the `copy_from_slice` can do something like `memmove`, and it would be better than pushing values one by one.

Reply:
Yeah, agree with your logic here. But at the same time, I think there is (a rather slight) chance that LLVM could recognize pattern and turn it into `mem{cpy,move}`. If that's the case, then I would rather keep the code simple.